controller/src/test/java/org/jboss/as/controller/persistence/PersistanceResourceTestCase.java

```diff
@@ -370,6 +371,17 @@ public void testRelativePathFromOutsideConfigDirectory() throws Exception {
         }
     }
 
+    @Test
+    public void testSymbolicLinkForConfigurationFiles() throws Exception {
+        Path file = Paths.get(createFile(externalDir, "standard.xml", "test").toString());
+        Path symbolicLink = standardDir.toPath().resolve("standard.xml");
+        if(Files.exists(symbolicLink)){
+            Files.delete(symbolicLink);
+        }
+        Files.createSymbolicLink(symbolicLink, file);
```

Comment:
You have to use absolute paths to create the symbolicLink, otherwise, the link will be created by using the current relative paths and will end up in an invalid link.

Reply:
@parsharma You have missed this comment, you have to use absolute paths when creating the symbolic link.